ui/src/services/platformlibrary/metrics.js

```diff
@@ -501,6 +501,34 @@ export const getWorkloadPlaneBandWidthAvgOutQuery = (
   };
 };
 
+// this query is to get the bandwidth for all interfaces eth0, eth1 and eth2
+// then we do filter base on the interface of the node which we can retrieve from Salt API
+export const getNodesPlanesBandwidthInQuery = (
+  timespanProps,
+  devices: string,
+) => {
+  const nodesPlanesBandwidthInQuery = `avg(irate(node_network_receive_bytes_total{device=~"${devices}"}[5m])) by (instance,device)`;
```

Comment:
I think it is the responsability of this function to join devices with pipes. If we have to evlove the query later on we might just want to have a list of unique devices as an argument and build the query in another way given this list (e.g swithing the query to cluster average...)